### PR TITLE
Create modal to input title description for an iteration.

### PR DESCRIPTION
**Pull request by vikram-raj**
Fix: https://github.com/fabric8io/fabric8-planner/issues/720

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
[test]

File: src/app/iteration/iteration.component.html [hunk 1]
```diff
@@ -187,8 +187,30 @@
...
+          <label for="iteration-name">Name <span class="field-required-icon">*</span></label>
+          <input id="iteration-name" type="text" class="form-control" placeholder="Enter a unique iteration name">
+          <span class="help-block">Iteration names must be unique within a project</span>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="iteration-description">Description</label>
+        <textarea class="form-control" id="iteration-description" rows="3"></textarea>
+      </div>
+      <div class="text-right">
```
Comment (SMahil) [inline src/app/iteration/iteration.component.html]: this block has to come under modal-footer

File: src/app/iteration/iteration.component.html [hunk 2]
```diff
@@ -187,8 +187,30 @@
...
 <!-- Iteration End-->
+
+<modal #createIterationDialog title="Create Iteration">
+  <modal-content>
+    <div class="col-md-12 col-sm-12 col-xs-12 createProjectBlock">
+      <div class="form-group">
+        <div class="search-pf-input-group">
+          <label for="iteration-name">Name <span class="field-required-icon">*</span></label>
+          <input id="iteration-name" type="text" class="form-control" placeholder="Enter a unique iteration name">
+          <span class="help-block">Iteration names must be unique within a project</span>
```
Comment (SMahil) [inline src/app/iteration/iteration.component.html]: if this is important note i think it should come in different colour

**Comment by SMahil** [comment]
@vikram-raj Transition is not working. You have to add it in .ts file as we did in delete modal

**Pull request merged**